imports/api/accounts/methods.js

```diff
@@ -67,6 +67,16 @@ export const update = new ValidatedMethod({
     }
 });
 
+Meteor.methods({
```

Comment:
@kashifsulaiman @SyedAsherAhmed check old methods and see how we write code for methods.